tests/test_api.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import requests_mock
```

Comment:
Good catch. Just switched to tox to manage testing, so all should be good now. 👍